packages/manager/modules/pci/src/projects/project/notebooks/components/configuration-command/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pci_notebook_configuration_command_description": "With the following command, you can create the same notebookusing our ovhai CLI:",
+  "pci_notebook_configuration_command_help": "For more information, you can read the",
+  "pci_notebook_configuration_command_help_link": "documentation about ovhai CLI",
```

Comment:
This sentence should not be split as is. It will be a mess for translate team. So maybe the whole sentence should be a link ?

Reply:
Yeah... But isn't the sentence too long to be put in as a link? :D

Reply:
Yes it would be a bit long, but still acceptable IMHO
Also it is the cleaner solution, in regards of translations/tracking matters.

Reply:
ok point taken. The sentence has been merged into one :)
thanks